Index: Tech_Hub/obj/rider.project.restore.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>17027571307687013
===================================================================
diff --git a/Tech_Hub/obj/rider.project.restore.info b/Tech_Hub/obj/rider.project.restore.info
--- a/Tech_Hub/obj/rider.project.restore.info	
+++ b/Tech_Hub/obj/rider.project.restore.info	
@@ -1,1 +1,1 @@
-17027571307687013
\ No newline at end of file
+17030126897052178
\ No newline at end of file
